admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.js

```diff
@@ -181,6 +242,9 @@ export default class CombinationsManager {
    * @private
    */
   findCombinationId(input) {
-    return $(input).closest('tr').find('.combination-id-input').val();
+    return $(input)
+      .closest('tr')
+      .find('.combination-id-input')
```

Comment:
```suggestion
      .find('.combination-id-input')
```

Move selector to the object map file